package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.8.4",
+    "brace": "^0.11.1",
```

Comment:
bump plugin version # also in this file

Reply:
So from now on any code change should cause the version number to increase 0.0.0.1?

Reply:
@ftianli-amzn Any time you release a version that version should never change. Any new additions goes into the next version, in this case 1.9.0.1 as 1.9.0.0 was already released for this plugin. If we now release this under 1.9.0.1 and we need to do another change after and release it on this branch it would become 1.9.0.2

Reply:
> @ftianli-amzn Any time you release a version that version should never change. Any new additions goes into the next version, in this case 1.9.0.1 as 1.9.0.0 was already released for this plugin. If we now release this under 1.9.0.1 and we need to do another change after and release it on this branch it would become 1.9.0.2

Thank you! I see.